openshift/scripts/oc_deploy_to_production.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -l set -ex
```

Comment:
`set -ex` is a command that needs to run, not a comment

Reply:
Thanks I didn't realize. I've also seen deployments not complete and the pipeline turn green, I guess the exit code is not being set by the `oc` tool?